services/web/server/src/simcore_service_webserver/users/_notifications_handlers.py

```diff
@@ -38,16 +39,19 @@ async def _get_user_notifications(
     redis_client: aioredis.Redis, user_id: int, product_name: str
 ) -> list[UserNotification]:
     """returns a list of notifications where the latest notification is at index 0"""
-    raw_notifications: list[str] = await redis_client.lrange(
+    lrange = redis_client.lrange(
         get_notification_key(user_id), -1 * MAX_NOTIFICATIONS_FOR_USER_TO_SHOW, -1
     )
+    raw_notifications: list[str] = (
+        await lrange if isinstance(lrange, Awaitable) else lrange
+    )
```

Comment:
we use it async so we know it is an awaitable right? why use the if here and not just cast or ignore?